EBTExtractor.xcodeproj/project.pbxproj

```diff
@@ -807,13 +807,184 @@
 			};
 			name = Release;
 		};
+		616AF7791FD61DB3000FF69F /* Testing */ = {
```

Comment:
Can this configuration be called "Test" for verbiage consistency between "Debug" and "Release"?

Reply:
yup